tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -296,22 +311,30 @@ Then('it should not be possible to create files using the webUI', function () {
   })
 })
 
-When('the user renames file/folder {string} to {string} using the webUI', function (fromName, toName) {
-  return client.page.FilesPageElement.filesList().renameFile(fromName, toName)
```

Comment:
you can leave this kind of functions in the filesList object, only the details how to achieve it and the selectors are moved into the lower level page object

Reply:
done!..